lisa/testsuite.py

```diff
@@ -290,7 +290,18 @@ def _send_result_message(self, stacktrace: Optional[str] = None) -> None:
             # force refresh information, when test result status is changed. The
             # refreshed information is not used so far. But in case it's needed
             # in future, keep it up to date.
-            self._environment_information = self.environment.get_information()
+            # avoid fetching information unnecessarily when rerun env
+            force_run = True
+            if (
```

Comment:
The `if` block needs more explanation to understand what's the problem is fixed. so it won't be confused in future.